### ISSUE TITLE
Add autofix to font-weight-notation

### ISSUE THREAD

**Issue by hiroppy** [stylelint/stylelint#3158]
I read https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#adding-an-option-to-an-existing-rule and write this issue.

I already have written code.
see https://github.com/abouthiroppy/stylelint/commit/297723031bdcb239cf131fc90842a4d88757038e

### numeric
```
a { font-weight: normal; } -> a { font-weight: 400; }
a { font: italic bold 20px; } -> a { font: italic 700 20px; }
```

### named-where-possible
```
a { font-weight: 400; } -> a { font-weight: normal; }
a { font: italic 700 20px; } -> a { font: italic bold 20px; }
```

If this request is accepted, I will submit PR :)

**Comment by evilebottnawi** [comment]
@abouthiroppy PR welcome!

**Comment by jeddy3** [comment]
Fixing https://github.com/stylelint/stylelint/issues/3855#issuecomment-844116177 is a prerequisite to adding autofix.

See also: https://github.com/stylelint/stylelint/pull/3159#issuecomment-855271082

**Comment by FloEdelmann** [comment]
@hiroppy Sorry for the ping in this old issue, but given you've already written the code for the autofix, would you fancy opening a PR?

I think @jeddy3's comment is already resolved (#3855 was closed by #6005). I don't know if that requires changes in your code though.

**Comment by ybiquitous** [comment]
@FloEdelmann Thanks for the reminder. I'm going to work on this. 💪🏼
